packages/is-shallow-equal/package.json

```diff
@@ -20,11 +20,12 @@
 		"url": "https://github.com/WordPress/gutenberg/issues"
 	},
 	"files": [
-		"arrays.js",
-		"index.js",
-		"objects.js"
+		"lib",
```

Comment:
@aduth, isn’t it a breaking change to rename files? I recall you could use file imports to use low level functions. Was it documented or I made that up?

I’m not against the change, I like it but we should add CHANGELOG entry if it’s confirmed.

Reply:
Related previous discussion: https://github.com/WordPress/gutenberg/pull/18942#discussion_r395842392

Do you recall where said usage was promoted? I have vague memory of it. The low-level functions are still available as named exports.

Reply:
Before this PR: https://github.com/WordPress/gutenberg/pull/12825
It’s safe to proceed but we need to update benchmarks:
https://github.com/WordPress/gutenberg/blob/a2884c129598a2f020757820a7e52a3d2920e596/packages/is-shallow-equal/benchmark/index.js#L19-20

Reply:
Nice catch, thanks! Addressed:

- 6ee66b50f8 Update is-shallow-equal tests
- a213ca951c Update is-shallow-equal benchmarks